vm/packer/templates/opencart/packer.in.json

```diff
@@ -0,0 +1,7 @@
+{
+  "license": "c2d-opencart",
+  "source_image_family": "debian-9",
```

Comment:
can we go for debian 10?

Reply:
Unfortunately no :( mysql recipes are not compatible with buster.